engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/bulk/content-hosts-bulk-errata-modal.controller.js

```diff
@@ -100,7 +100,7 @@ angular.module('Bastion.content-hosts').controller('ContentHostsBulkErrataModalC
                 selectedHosts = hostIds;
 
             formData.authenticityToken = $window.AUTH_TOKEN.replace(/&quot;/g, '');
-            formData.remoteAction = 'errata_install';
+            formData['remote_action'] = 'errata_install';
             formData.errata = errataIds.join(',');
             if (selectedHosts.included.ids) {
                 formData.hostIds = selectedHosts.included.ids.join(',');
```

Comment:
is formData.hostIds going to work as well?  (or authenticityToken?)

Reply:
I can't apply an erratum from the bulk host tool:

![errata](https://user-images.githubusercontent.com/429763/38201971-24a8dcd2-3668-11e8-8ae5-7e50dcc245dc.png)

Works on a single host though.

Reply:
> is formData.hostIds going to work as well? (or authenticityToken?)

What do you mean?  Should those also be underscore case?

Reply:
I believe so yes

Reply:
@jlsherrill @stbenjam updated.